File: BOJ/Java/단계별로_풀어보기/B_01_입출력과_사칙연산/BOJ_10430.java
package 단계별로_풀어보기.B_01_입출력과_사칙연산;

/**
 * BOJ 10430 : 나머지
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.util.Scanner;

public class BOJ_10430 {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);

		double A = in.nextInt();
		double B = in.nextInt();
		double C = in.nextInt();

		System.out.println((A + B) % C);
		System.out.println((A % C + B % C) % C);
		System.out.println((A * B) % C);
		System.out.println((A % C * B % C) % C);
		in.close();
	}

}

File: BOJ/CPP/단계별로_풀어보기/입출력과_사칙연산/BOJ_10430.cpp
/*
    BOJ 10430 : 나머지

    author : kdgyun
    link https://st-lab.tistory.com
    link https://github.com/kdgyun
    Created by kdgyun
    Copyright © 2021 kdgyun. All rights reserved.
*/

#include <stdio.h>

int main(int argc, char const *argv[]) {
    int A, B, C;
    scanf("%d %d %d", &A, &B, &C);

    printf("%d\n", (A + B) % C);
    printf("%d\n", (A % C + B % C) % C);
    printf("%d\n", (A * B) % C);
    printf("%d\n", (A % C * B % C) % C);
    return 0;
}
